chromium/scripts/getpqc.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+mkdir -p $PROJECT
+cd $PROJECT
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+time fetch --nohooks chromium
```

Comment:
Mights (output from) the `time` commands confuse users?

Reply:
Yes. Will remove it.